tests/opentelemetry-docker-tests/tests/redis/test_redis_functional.py

```diff
@@ -209,7 +208,6 @@ def tearDown(self):
     def _check_span(self, span, name):
         self.assertEqual(span.name, name)
         self.assertIs(span.status.status_code, trace.StatusCode.UNSET)
-        self.assertEqual(span.attributes.get(SpanAttributes.DB_NAME), 0)
```

Comment:
same as above

Reply:
Yes, that makes sense. Will add it and commit.

Reply:
Ok, done.